loader/icd_dispatch_generated.c

```diff
@@ -4246,7 +4246,7 @@ CL_API_ENTRY void CL_API_CALL clSVMFree(
 {
 #if defined(CL_ENABLE_LAYERS)
     if (khrFirstLayer)
-        return khrFirstLayer->dispatch.clSVMFree(
+        khrFirstLayer->dispatch.clSVMFree(
```

Comment:
I guess the `return` is cut because `clSVMFree` returns `void`.

But now the function will continue on and call `context->dispatch->clSVMFree`.   Looking at `clCreateSamplerWithProperties` for comparison (just because it happens to be just below here), it does **not** call the `context->dispatch->clFoo` for this case.

Should there be a `return;` after this line (but adding `{}` to the if-block of course)?

Reply:
Yes I have fixed the generators but I will suggest a fix similar to the generated files here.

Reply:
The pull request is there: https://github.com/KhronosGroup/OpenCL-ICD-Loader/pull/139.

Reply:
Ah, thank you @Kerilk, I hadn't realized that this file was generated.

Reply:
Is it okay if the generated file remains patched and the PR will overwrite it in the future?

(Ultimately we ought to generate those files on every build node; one `add_custom_target()` to the CMake scripts isn't black magic. How does one use mako to generate those headers?)

Reply:
But indeed, it was cut because initially I was set on making `-Werror` and `/WX` happy, but ultimately I gave up. One of the tested compilers was warning about returning `void`.

Reply:
https://github.com/KhronosGroup/OpenCL-ICD-Loader/pull/139 has been merged, so maybe reverting those two (`icd_dispatch_generated.c` and `icd_print_layer_generated.c`) would avoid a conflict.